rust/blockstore/src/arrow/flusher.rs

```diff
@@ -8,6 +8,7 @@ use chroma_error::ChromaError;
 use futures::{StreamExt, TryStreamExt};
 use uuid::Uuid;
 
+#[derive(Clone)]
```

Comment:
we must make flushers clonable because `.flush()` consumes `self` but operators receive `&input`
I don't love this, I think the alternative is allowing operators to consume the input